verifier.sh

```diff
@@ -348,18 +349,31 @@ _devtest_innervm_run () {
     ssh -t  $lxc_ip "rm -f ssh.log"
 
     ssh -t  $lxc_ip "sudo apt-get update"
+    # use this parameter to avoid blocks with sudoers updates: '-o Dpkg::Options::=--force-confdef'
     ssh -t  $lxc_ip "sudo apt-get -y upgrade"
+    ssh -t  $lxc_ip "wget http://ftp.openquake.org/mirror/mozilla/geckodriver-latest-linux64.tar.gz ; tar zxvf geckodriver-latest-linux64.tar.gz ; sudo cp geckodriver /usr/local/bin"
+    ssh -t  $lxc_ip "sudo pip install -U selenium==3.0.1"
```

Comment:
Old version of selenium is already installed on the LXC ephemeral container so we can keep it in mind and pay attention about possible related problems.